homeassistant/components/dte_energy_bridge/sensor.py

```diff
@@ -104,6 +104,18 @@ def update(self):
             return
 
         val = float(response_split[0])
+		
+        # A workaround for a bug in the DTE energy bridge.
```

Comment:
These quirks should be handled by a 3rd party library. We won't accept more changes to this integration until the device specific interface code and data parsing has been moved to a standalone library published on PyPI.

https://developers.home-assistant.io/docs/creating_component_code_review#4-communication-with-devicesservices